torchaudio/pipelines/_wav2vec2/impl.py

```diff
@@ -60,7 +78,7 @@ def _get_state_dict(self, dl_kwargs):
         state_dict = load_state_dict_from_url(url, **dl_kwargs)
         return state_dict
 
-    def get_model(self, *, dl_kwargs=None) -> Wav2Vec2Model:
+    def get_model(self, *, dl_kwargs=None) -> _Wav2Vec2Model:
```

Comment:
Can we make it to generic `torch.nn.Module`, then mention that this method returns a variation of `Wav2Vec2Model` in the docstring so that users browsing the documentation page can go to the model definition easily?

To link to the model definition with Sphinx syntax, please use ```:py:class:`~torchaudio.models.Wav2Vec2` ```.

---
Follow-up: We should also do the same for SourceSeparationBundle, which returns different models https://pytorch.org/audio/stable/generated/torchaudio.pipelines.SourceSeparationBundle.html#get-model